[Bug]:

### Frontend Version

Master

### Expected Behavior

No error when commiting

### Actual Behavior

fail pre-submit for git commit

### Steps to Reproduce

Create a new branch
Add space to a file and commit
Run git commit

### Debug Logs

```powershell
✗ git commit -m "test"
✔ Preparing lint-staged...
⚠ Running tasks for staged files...
  ❯ package.json — 1 file
    ✔ ./**/*.{js,ts,tsx,vue} — 1 file
    ❯ **/*.ts — 1 file
      ✖ tsc-files --noEmit [FAILED]
↓ Skipped because of errors from tasks.
✔ Reverting to original state because of errors...
✔ Cleaning up temporary files...

✖ tsc-files --noEmit:
src/extensions/core/groupNodeManage.ts(108,5): error TS2741: Property 'nodeData' is missing in type 'typeof LGraphNode' but required in type 'LGraphNodeConstructor<LGraphNode>'.
src/extensions/core/rerouteNode.ts(31,14): error TS2339: Property 'onAfterGraphConfigured' does not exist on type 'RerouteNode'.
src/extensions/core/rerouteNode.ts(222,14): error TS2339: Property 'isVirtualNode' does not exist on type 'RerouteNode'.
src/extensions/core/uploadAudio.ts(98,16): error TS2551: Property 'addDOMWidget' does not exist on type 'LGraphNode'. Did you mean 'addWidget'?
src/extensions/core/uploadAudio.ts(107,35): error TS2551: Property 'onExecuted' does not exist on type 'LGraphNode'. Did you mean 'onExecute'?
src/extensions/core/uploadAudio.ts(108,16): error TS2551: Property 'onExecuted' does not exist on type 'LGraphNode'. Did you mean 'onExecute'?
src/extensions/core/uploadAudio.ts(170,40): error TS2339: Property 'onGraphConfigured' does not exist on type 'LGraphNode'.
src/extensions/core/uploadAudio.ts(171,14): error TS2339: Property 'onGraphConfigured' does not exist on type 'LGraphNode'.
src/extensions/core/webcamCapture.ts(58,31): error TS2551: Property 'addDOMWidget' does not exist on type 'LGraphNode'. Did you mean 'addWidget'?
src/extensions/core/widgetInputs.ts(34,10): error TS2339: Property 'isVirtualNode' does not exist on type 'PrimitiveNode'.
src/extensions/core/widgetInputs.ts(116,34): error TS2339: Property 'widgets_values' does not exist on type 'PrimitiveNode'.
src/extensions/core/widgetInputs.ts(239,32): error TS2339: Property 'widgets_values' does not exist on type 'PrimitiveNode'.
src/extensions/core/widgetInputs.ts(250,25): error TS2339: Property 'widgets_values' does not exist on type 'PrimitiveNode'.
src/extensions/core/widgetInputs.ts(252,9): error TS2578: Unused '@ts-expect-error' directive.
src/hooks/bottomPanelTabs/integratedTerminalTab.ts(3,32): error TS2307: Cannot find module '@/components/bottomPanel/tabs/IntegratedTerminal.vue' or its corresponding type declarations.
src/hooks/sidebarTabs/modelLibrarySidebarTab.ts(3,36): error TS2307: Cannot find module '@/components/sidebar/tabs/ModelLibrarySidebarTab.vue' or its corresponding type declarations.
src/hooks/sidebarTabs/nodeLibrarySidebarTab.ts(3,35): error TS2307: Cannot find module '@/components/sidebar/tabs/NodeLibrarySidebarTab.vue' or its corresponding type declarations.
src/hooks/sidebarTabs/queueSidebarTab.ts(4,29): error TS2307: Cannot find module '@/components/sidebar/tabs/QueueSidebarTab.vue' or its corresponding type declarations.
src/hooks/sidebarTabs/workflowsSidebarTab.ts(5,33): error TS2307: Cannot find module '@/components/sidebar/tabs/WorkflowsSidebarTab.vue' or its corresponding type declarations.
src/scripts/app.ts(1568,18): error TS2551: Property 'onExecuted' does not exist on type 'LGraphNode'. Did you mean 'onExecute'?
src/scripts/app.ts(1568,35): error TS2551: Property 'onExecuted' does not exist on type 'LGraphNode'. Did you mean 'onExecute'?
src/scripts/app.ts(1667,14): error TS2339: Property 'onGraphConfigured' does not exist on type 'LGraphNode'.
src/scripts/app.ts(1674,14): error TS2339: Property 'onAfterGraphConfigured' does not exist on type 'LGraphNode'.
src/scripts/app.ts(2092,20): error TS2576: Property 'comfyClass' does not exist on type 'ComfyNode'. Did you mean to access the static member 'ComfyNode.comfyClass' instead?
src/scripts/domWidget.ts(283,22): error TS2551: Property 'addDOMWidget' does not exist on type 'LGraphNode'. Did you mean 'addWidget'?
src/services/dialogService.ts(5,33): error TS2307: Cannot find module '@/components/dialog/content/LoadWorkflowWarning.vue' or its corresponding type declarations.
src/services/dialogService.ts(6,34): error TS2307: Cannot find module '@/components/dialog/content/MissingModelsWarning.vue' or its corresponding type declarations.
src/services/dialogService.ts(7,34): error TS2307: Cannot find module '@/components/dialog/content/SettingDialogContent.vue' or its corresponding type declarations.
src/services/dialogService.ts(8,33): error TS2307: Cannot find module '@/components/dialog/header/SettingDialogHeader.vue' or its corresponding type declarations.
src/services/dialogService.ts(10,41): error TS2307: Cannot find module '@/components/dialog/content/ExecutionErrorDialogContent.vue' or its corresponding type declarations.
src/services/dialogService.ts(11,38): error TS2307: Cannot find module '@/components/templates/TemplateWorkflowsContent.vue' or its corresponding type declarations.
src/services/dialogService.ts(12,33): error TS2307: Cannot find module '@/components/dialog/content/PromptDialogContent.vue' or its corresponding type declarations.
husky - pre-commit script failed (code 1)
```
```


### Browser Logs

NA

### Setting JSON

NA

### What browsers do you use to access the UI ?

_No response_

### Other

_No response_

--- a/.husky/pre-commit
+++ b/.husky/pre-commit
@@ -1,5 +1,2 @@
-if [[ "$OS" == "Windows_NT" ]]; then
-  npx.cmd lint-staged
-else
-  npx lint-staged
-fi
+npx lint-staged
+npm run typecheck

--- a/package.json
+++ b/package.json
@@ -58,7 +58,6 @@
     "tailwindcss": "^3.4.4",
     "ts-jest": "^29.1.4",
     "ts-node": "^10.9.2",
-    "tsc-files": "^1.1.4",
     "tsx": "^4.15.6",
     "typescript": "^5.4.5",
     "typescript-eslint": "^8.0.0",
@@ -94,7 +93,9 @@
     "zod-validation-error": "^3.3.0"
   },
   "lint-staged": {
-    "./**/*.{js,ts,tsx,vue}": "prettier --write",
-    "**/*.ts": "tsc-files --noEmit"
+    "./**/*.{js,ts,tsx,vue}": [
+      "prettier --write",
+      "git add"
+    ]
   }
 }

--- a/package.json
+++ b/package.json
@@ -94,8 +94,7 @@
   },
   "lint-staged": {
     "./**/*.{js,ts,tsx,vue}": [
-      "prettier --write",
-      "git add"
+      "prettier --write"
     ]
   }
 }

